### PR TITLE
Add php 8 compatibility

### PR DESCRIPTION
**Pull request by CMalvika**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,23 +30,23 @@
         }
     },
     "require-dev": {
-        "symfony/dom-crawler": "~3",
+        "symfony/dom-crawler": "~4 || ~5",
```
Comment (lstrojny) [inline composer.json]: Decide for one Symfony version constraint, so either all 4.x versions or 4.4

**Review by lstrojny — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (CMalvika) [reply composer.json]: True! Updated.

**Review by lstrojny — APPROVED** [review]

**Comment by lstrojny** [comment]
What often makes sense for CI on packages with wide dependency ranges is to add build variants where you composer  install with `--prefer-lowest`. This is the idea https://evertpot.com/testing-composer-prefer-lowest/

Out of scope for this PR but generally a good idea to add and fairly easy to do with the composer install action.

**Comment by CMalvika** [comment]
> add build variants



> What often makes sense for CI on packages with wide dependency ranges is to add build variants where you composer install with `--prefer-lowest`. This is the idea https://evertpot.com/testing-composer-prefer-lowest/
> 
> Out of scope for this PR but generally a good idea to add and fairly easy to do with the composer install action.

Thank you very much for the suggestion, I'll have a look.

**Pull request merged**